keps/sig-storage/20191125-bucket-provisioning.md

```diff
@@ -299,7 +314,7 @@ provisioner: [1]
 supportedProtocols: [2]
 accessMode: {"ro", "wo", "rw"} [3]
 releasePolicy: {"Delete", "Retain"} [4]
-objectBucketName: [5]
+bucketIdentifier: [5]
```

Comment:
Consider adding to `provisioner` below:
`Provisioner names follows the DNS naming conventions, eg. "ceph-rgw.cosi.redhat.com"`?

Reply:
Agree, but since it's appears in a few places (and likely will appear more once we start defining the gRPC interface), I'll call it out explicitly in one place